### DISCUSSION TITLE
[求助]: 导航菜单增加后出现错误

### DISCUSSION THREAD

**Discussion by ghost** [valor-x/hexo-theme-solitude#233] [Q&A]
### 前置确认

- [X] 我确认我运行的是最新版本的主题，并且安装了所需的依赖，在[FAQS](https://solitude-docs.efu.me/faq)中也未找到类似问题。

### ⚠️ 搜索issues中是否已存在类似问题

- [X] 我已经搜索过issues和disscussions，没有跟我遇到的问题相关的issue

### 操作系统类型?

Mac/Linux

### 运行的Hexo版本是?

Hexo 7.0.0

### 你所使用的主题版本是稳定/最新?

稳定版（main）

### 问题描述 😯

当配置导航菜单 nav.group. 后会出现与配置文件无关的填充项。

配置如下：

    Solitude: https://github.com/valor-x/hexo-theme-solitudexxxxx || /img/pwa/favicon.ico # 子菜单名称(submenu name): 链接地址(address) || 图标(icon)
    A: https://www.q.com || /img/pwa/favicon.ico

错误如图：
<img width="616" alt="image" src="https://github.com/valor-x/hexo-theme-solitude/assets/152959149/88c979d1-4f82-4fe2-b1c0-811f4aff0dd2">

**Comment by everfu** [comment]
<img width="415" alt="image" src="https://github.com/valor-x/hexo-theme-solitude/assets/74389842/731763b7-eda0-4371-9086-689463850949">
<img width="1191" alt="image" src="https://github.com/valor-x/hexo-theme-solitude/assets/74389842/f6699f6a-56d1-44a7-9a74-6d884e931faa">

**Comment by everfu** [comment]
没有问题，看你的配置是不是有错误

**Comment by everfu** [comment]
你看看配置信息，是不是配置错了？

**Comment by ghost** [comment]
> 你看看配置信息，是不是配置错了？

重新复制的配置文件，问题依旧，且在 Post 也会出现。

<img width="758" alt="image" src="https://github.com/valor-x/hexo-theme-solitude/assets/152959149/ec550b58-74b0-4e53-b65e-4e57d4582dd8">

**Comment by everfu** [comment]
配置文件nav信息全放出来

**Comment by ghost** [comment]
> 配置文件nav信息全放出来

```yaml
# 导航栏
# NavBar
nav:
  # logo 箱子
  # logo group
  group:
  # 项目: # 分类名称
    Solitude: https://github.com/valor-x/hexo-theme-solitude || /img/pwa/favicon.ico # 子菜单名称(submenu name): 链接地址(address) || 图标(icon)
```

**Comment by ghost** [comment]
> 配置文件nav信息全放出来

hexo 版本信息


```
hexo v
INFO  Validating config
INFO  
  ╭─────────────────────────────────────────────────────────────────────────────────────────────╮
  │                                                                                             │
  │      Theme: Solitude                                                                        │
  │      Version: v1.7.14                                                                        │
  │      Changelog: https://github.com/valor-x/hexo-theme-solitude/releases/tag/v1.7.14          │
  │                                                                                             │
  ╰─────────────────────────────────────────────────────────────────────────────────────────────╯
  
hexo: 7.1.1
hexo-cli: 4.3.1
os: darwin 23.4.0 14.4.1

node: 21.7.1
acorn: 8.11.3
ada: 2.7.6
ares: 1.27.0
base64: 0.5.2
brotli: 1.1.0
cjs_module_lexer: 1.2.2
cldr: 44.1
icu: 74.2
llhttp: 9.1.3
modules: 120
napi: 9
nghttp2: 1.60.0
openssl: 3.2.1
simdjson: 3.7.0
simdutf: 4.0.8
tz: 2023c
undici: 6.6.2
unicode: 15.1
uv: 1.48.0
uvwasi: 0.0.20
v8: 11.8.172.17-node.20
zlib: 1.2.12

```

**Comment by everfu** [comment]
<img width="563" alt="image" src="https://github.com/valor-x/hexo-theme-solitude/assets/74389842/1787daf7-8fae-466b-b869-9f3c444df9df">

你这写错了
```yaml
group:
    project:
       item:
```
这是三级，你把项目注释了

**Comment by everfu** [comment]
我帮你移动到 [Discussions](https://github.com/valor-x/hexo-theme-solitude/discussions) 了，这不是bug
